src/Tools/SyntaxVisualizer/Roslyn.SyntaxVisualizer.DgmlHelper/SyntaxDgmlHelper.vb

```diff
@@ -1,4 +1,4 @@
-' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Looks like the BOM might have changed? Can you make sure that this part of the file isn't touched.